app/models/project_mediaflux.rb

```diff
@@ -43,6 +44,16 @@ def self.project_values(project:)
     values
   end
 
+  def self.xml_payload(project:, xml_namespace: nil)
```

Comment:
@carolyncole In looking at this, I think this is returning the XML to create the namespace only, but the original code (https://github.com/pulibrary/tiger-data-app/blob/main/app/models/project_mediaflux.rb#L3-L21) sends two requests to Mediaflux: One to create the namespace and one to create the collection assets underneath. I am not sure how we want to represent this in the what will produce for people to reuse.

Reply:
@hectorcorrea correct.  It only creates the collection asset.create XML, not the namespace create XML.

Reply:
We could possible attach 3 files to the email, one for the collection, one for the namespace, and then the json representation of the data.